common/supplemental/units.xml

```diff
@@ -60,10 +102,10 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<convertUnit source='pound' target='kilogram' factor='lb2kg'/>
 		<convertUnit source='stone' target='kilogram' factor='lb2kg*14'/>
 		<convertUnit source='ton' target='kilogram' factor='lb2kg*2000'/>
-		<convertUnit source='horsepower' target='square-meter-kilogram-per-cubic-second' factor='ft2m * lb2kg * gravity * 550'/>
+		<convertUnit source='horsepower' target='kilogram-square-meter-per-cubic-second' factor='ft2m * lb2kg * gravity * 550'/>
 		<convertUnit source='inch-hg' target='kilogram-per-meter-square-second' factor='3386.389'/>
 		<convertUnit source='knot' target='meter-per-second' factor='1852/3600'/>
-		<convertUnit source='fahrenheit' target='celsius' factor='5/9' offset='-160/9'/>
+		<convertUnit source='fahrenheit' target='kelvin' factor='5/9' offset='-2618.35/9'/>
```

Comment:
I think this factor/offset may be incorrect. If Fahrenheit is `0` then if I've understood correctly the result will be `(0 * 5 / 9) - (-2618.35 / 9) ~= 291` when the correct result is `~= 255.3`.